icd/api/include/khronos/vulkan.h

```diff
@@ -53,6 +53,9 @@
 #ifdef setbit
 #undef setbit
 #endif
+#ifdef DestroyAll
+#undef DestroyAll
+#endif
```

Comment:
We don't want to modify the khronos headers.

Reply:
The description of this file says `Proxy to the real Khronos Vulkan header.`, so I assume this is not the khronos header

Reply:
I guess that is our wrapper to the khronos headers.